src/misc/inc/1DQuadrature.h

```diff
@@ -44,8 +45,8 @@ namespace QUESO {
 
     Base class for numerical integration via quadrature rules of one-dimensional functions.
 */
-class
-Base1DQuadrature {
+class Base1DQuadrature : public QuadratureBase
```

Comment:
Actually, I notice you didn't add this class, it was already in QUESO.  So adding a pure virtual function would be a breaking change.